libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -107,7 +108,7 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
 
     // @Param{Copter, Plane, Sub}: ALT_MIN
     // @DisplayName: Fence Minimum Altitude
-    // @Description: Minimum altitude allowed before geofence triggers
+    // @Description: Minimum altitude allowed before geofence triggers. See FENCE_ALT_TYPE for reference frame.
```

Comment:
Then ditto "See FENCE_ALT_MIN_TP for reference frame."

Reply:
Fixed